### DISCUSSION TITLE
Is it possible to close/reopen listing through the listing API? / how to find out the right values for each prop?

### DISCUSSION THREAD

**Discussion by PhenomEY** [amzn/selling-partner-api-models#3350] [General]
Hello,

i was wondering if its possible to close a listing temporarily through the listing API.
The property "fulfillment_channel_availability" sounds interesting, but i cant find any documentation about that one.

How to find out which property expects which value?
for example: 
"item_name" requires a "language_tag"
"purchasable_offer" requires "currency" and "our_price"

Where can i find out the needed values for all those different props?

**Comment by chapmanjw** [comment]
@PhenomEY You can use the Product Type Definitions API to get the JSON Schemas describing the attributes and their requirements for a given product type: https://developer-docs.amazon.com/sp-api/docs/product-type-api-use-case-guide
